tests/playbooks/tests_ethtool_features.yml

```
@@ -145,6 +145,10 @@
                       features:
                         tx_tcp_segmentation: "no"
                         tx-tcp-segmentation: "no"
+                __header: "{{ __lsr_ansible_managed }}"
+              vars:
+                __lsr_ansible_managed: "{{
+                  lookup('template', 'get_ansible_managed.j2') }}"
```

```suggestion
                __header: "# Anisble managed test header"
```

Please consider to just using some placeholder header here since this test call of the internal module does not need to use the ansible header - this can simplify the test a little bit.

---

Done. Thanks.